packages/api-client/README.md

```diff
@@ -24,27 +32,35 @@ yarn add @opendatasoft/api-client
 
 ## Get started
 
-Here is a quick example to get you started :
+Here is a quick example to get you started:
 
 ```javascript
 import { ApiClient, fromCatalog } from '@opendatasoft/api-client';
 
-const client = new ApiClient({ domain: 'public' });
-const query = fromCatalog().facets();
+// Here we initialize the Client by indicating the domain to request.
+const client = new ApiClient({ domain: "public" });
 
-client.get(query)
-      .then((response) => console.log(response))
-      .catch((error) => console.error(error));
+const query = fromCatalog() // From the domain catalog
+    .dataset("worldcitiespop") // ... we'll use the dataset "worldcitiespop"
+    .aggregates() // ... in order to make an aggregation.
+    .where("country:'fr'") // Filtering records where country === "fr".
+    .groupBy("city, population") // Selecting the fields "city" and "population".
+    .orderBy("-population") // Sorted by the most populated cities.
```

Comment:
I think it's clearer to mention asc/desc order where relevant. I changed the verb from "Sorted" to "Sort" to use the same verb form everywhere.
```suggestion
    .orderBy("-population") // Sort by population in descending order.
```